src/plugins/telemetry/include/plugins/telemetry/telemetry.h

```diff
@@ -148,6 +148,24 @@ class Telemetry : public PluginBase {
      */
     friend std::ostream& operator<<(std::ostream& str, Telemetry::LandedState const& landed_state);
 
+    /**
+     * @brief VTOL State enumeration
+     */
+    enum class VtolState {
```

Comment:
I used the same naming scheme like in the other enums although I would personally prefer using more intuitive names than Mc and Fw

Reply:
Yes the enums are just based on MAVLink definitions https://mavlink.io/en/messages/common.html#MAV_VTOL_STATE